spec.md

```diff
@@ -80,85 +80,97 @@ A CO SHOULD be equipped to handle both centralized and headless plugins, as well
 Several of these possibilities are illustrated in the following figures.
 
 ```
-                             CO "Master" Host
-+-------------------------------------------+
-|                                           |
-|  +------------+           +------------+  |
-|  |     CO     |   gRPC    | Controller |  |
-|  |            +----------->   Plugin   |  |
-|  +------------+           +------------+  |
-|                                           |
-+-------------------------------------------+
-
-                            CO "Node" Host(s)
-+-------------------------------------------+
-|                                           |
-|  +------------+           +------------+  |
-|  |     CO     |   gRPC    |    Node    |  |
-|  |            +----------->   Plugin   |  |
-|  +------------+           +------------+  |
-|                                           |
-+-------------------------------------------+
+                                   CO "Master" Host
++-------------------------------------------------+
+|                                                 |
+|  +------------+           +------------------+  |
+|  |     CO     |   gRPC    |   Controller     |  |
+|  |            +--+-------->     Plugin       |  |
+|  +------------+  |        +------------------+  |
+|                  |                              |
+|                  |        +------------------+  |
+|                  |        | Group Controller |  |
+|                  +-------->     Plugin       |  |
+|                           +------------------+  |
+|                                                 |
++-------------------------------------------------+
+
+                                  CO "Node" Host(s)
++-------------------------------------------------+
+|                                                 |
+|  +------------+           +------------------+  |
+|  |     CO     |   gRPC    |       Node       |  |
+|  |            +----------->      Plugin      |  |
+|  +------------+           +------------------+  |
+|                                                 |
++-------------------------------------------------+
 
 Figure 1: The Plugin runs on all nodes in the cluster: a centralized
-Controller Plugin is available on the CO master host and the Node
-Plugin is available on all of the CO Nodes.
+Controller Plugin and Group Controller Plugin (optional) are available
```

Comment:
"Control Plugin" is also optional.
I'd suggest change "(optional)" to "(alpha)".